XenAdmin/Wizards/PatchingWizard/PatchingWizard.cs

```diff
@@ -339,6 +339,7 @@ private void RemoveDownloadedPatches()
             else
             {
                 listOfDownloadedFiles.AddRange(PatchingWizard_UploadPage.AllDownloadedPatches.Values);
+                listOfDownloadedFiles.AddRange(PatchingWizard_SelectPatchPage.UnzippedUpdateFiles);

```

Comment:
I think that we need to add the unzip files regardless of the mode the wizard is in when the cleanup is called, because I can, for example, select a zip file and then select the automated mode; if I finish (or cancel) the wizard in the automated mode, the unzipped file won't be cleaned up.
I haven't tried it out, but I believe this is the case.

We should also evaluate if we need to do the same with the downloaded patches (line 341 above).